docs/release-notes.md

```diff
@@ -1,3 +1,20 @@
+## Release v1.7.0
+### Date: May 30, 2024
+
+Release Manager: Russ Poetker, JHU
+
+This release focused on adding a GitHub workflow that will complete the release of all PASS components. The pass-core metadata schema service was changed as a first step in supporting InvenioRDM integration. A few smaller bugs were also fixed.
```

Comment:
I'd suggest adding a sentence or two here about the documentation work that was completed during this release (doc review manual, style guide, first-round of review). Maybe also note the background work happening to transition to Terraform.

Reply:
Agreed. That makes sense.

Reply:
Done